content/blog/email-alias-vs-plus-sign.md

```diff
@@ -57,6 +57,26 @@ Email addresses that contain the plus sign are sometimes (incorrectly) considere
 
 If you use Gmail, you can't also reply from the subaddress. When you reply to an email sent to a `name+newsletter@gmail.com`, the reply will come from your real email address `name@gmail.com`
 
+{{< rawhtml >}}
+<br>
+  <div style="padding: 35px; background-color: #f8f9fa; border-radius: 15px;">
+<!--     <img src="/logo.svg" style="height:35px;"/>
+    <br><br><br> -->
+    <p class="mb-2 h5"><strong>Regain control of your inbox with SimpleLogin</strong></p>
+    <p class="mb-2">
+        SimpleLogin offers a better protect your identity while browsing online then subaddressing. With SimpleLogin you can generate a unique <em data-toggle="tooltip" data-html="true" title="" data-original-title="An email alias (or alias for short) is an email address that <b>doesn't store</b> emails: all emails sent to an alias are forwarded to your personal email.">email aliases</em> that <strong>forwards instantly</strong> all incoming emails to your primary email address.
```

Comment:
Small correction: "SimpleLogin offers a better protection for your identity than subaddressing. With SimpleLogin you can generate a unique email alias that forwards instantly all incoming emails to your inbox."